packages/ui-components/src/lib/index.ts

```diff
@@ -46,7 +46,9 @@ export { default as IconSuccess } from './components/IconSuccess.svelte';
 export { default as IconTelegram } from './components/IconTelegram.svelte';
 export { default as IconWalletConnect } from './components/IconWalletConnect.svelte';
 export { default as IconWarning } from './components/IconWarning.svelte';
-
+export { default as VaultBalanceChangesTable } from './components/tables/VaultBalanceChangesTable.svelte';
+export { default as VaultBalanceChart } from './components/charts/VaultBalanceChart.svelte';
+export { default as VaultDetail } from './components/detail/VaultDetail.svelte';
 //Types
 export type { AppStoresInterface } from './types/appStores.ts';
 export type { ConfigSource, OrderbookConfigSource, OrderbookRef } from './typeshare/config';
```

Comment:
typehsare - might need to leave this one until we have new Yaml config structs done